automation/Extension/webext-instrumentation/package.json

```diff
@@ -20,7 +20,6 @@
     "fix:prettier": "prettier \"src/**/*.ts\" --write",
     "fix:tslint": "tslint -t verbose --fix --project .",
     "test": "run-s build test:*",
-    "test:lint": "tslint -t verbose --project . && prettier \"src/**/*.ts\" --list-different",
```

Comment:
Why remove this?

Reply:
@nhnt11

Reply:
no reason to remove it, if this came from my original patch then it's likely just because I wanted to speed up my workflow when building.

Reply:
Reverted